bertie_bogo_shim/src/main.rs

```diff
@@ -0,0 +1,214 @@
+// # Bertie BoGo Shim
```

Comment:
Doing `cargo run` (obviously) gives me just an error.
Please add a Readme that describes how to use this.